ush/drive_makeprepbufr.sh

```diff
@@ -1,142 +0,0 @@
-#! /usr/bin/env bash
```

Comment:
Why is this script being removed? Is it related to nemsio somehow?

Reply:
This is a hardwired driver only configured for nemsio. Therefore, it is removed.